modules/ossm-rn-known-issues.adoc

```diff
@@ -15,30 +15,47 @@ Module included in the following assemblies:
 
 ifndef::openshift-rosa[]
-* The Bookinfo sample application cannot be installed on IBM Z and IBM Power.
+* The Bookinfo sample application cannot be installed on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
-* WebAssembly extensions are not supported on IBM Z and IBM Power.
+* WebAssembly extensions are not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
-* LuaJIT is not supported on IBM Power.
+* LuaJIT is not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 
+* Single stack IPv6 support is not available on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
 endif::openshift-rosa[]
+
 [id="ossm-rn-known-issues-ossm_{context}"]
 == {SMProductShortName} known issues
 
 These are the known issues in {SMProductName}:
 
-* https://issues.redhat.com/browse/OSSM-2221[OSSM-2221] Gateway injection does not work in control plane namespace. If you use the Gateway injection feature to create a gateway in the same location as the control plane, the injection fails and OpenShift generates this message:
+* https://issues.redhat.com/browse/OSSM-3890[OSSM-3890] Attempting to use the Gateway API in a multitenant mesh deployment generates an error message similar to the following:
++
+[source,text]
+----
+2023-05-02T15:20:42.541034Z	error	watch error in cluster Kubernetes: failed to list *v1alpha2.TLSRoute: the server could not find the requested resource (get tlsroutes.gateway.networking.k8s.io)
+2023-05-02T15:20:42.616450Z	info	kube	controller "gateway.networking.k8s.io/v1alpha2/TCPRoute" is syncing...
+----
++
+To support Gateway API in a multitenant mesh deployment, all Gateway API Custom Resource Definition (CRD) files must be present in the cluster. 
 +
-`Warning  Failed          10s   kubelet, ocp-wide-vh8fd-worker-vhqm9  Failed to pull image "auto": rpc error: code = Unknown desc = reading manifest latest in docker.io/library/auto: errors`
+In a multitenant mesh deployment, CRD scan is disabled, and Istio has no way to discover which CRDs are present in a cluster. As a result, Istio attempts to watch all supported Gateway API CRDs, but generates errors if some of those CRDs are not present.
 +
-To create a gateway in the control plane namespace, use the `gateways` parameter in the SMCP spec to configure ingress and egress gateways for the mesh.
+{SMProductShortName} 2.3.1 and later versions support both `v1alpha2` and `v1beta1` CRDs. Therefore, both CRD versions must be present for a multitenant mesh deployment to support the Gateway API.
++
+Workaround: In the following example, the `kubectl get` operation installs the `v1alpha2` and `v1beta1` CRDs. Note the URL contains the additional `experimental` segment, and update any of your existing scripts accordingly:
```

Comment:
Sounds like it should be either:

Note the URL contains the additional `experimental` segment, and **updates** any ...

OR

Note the URL contains the additional `experimental` segment, and **you must** update any